src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
 // write code here
+document.addEventListener('DOMContentLoaded', () => {
+  const tree = document.querySelector('.tree');
+
+  // eslint-disable-next-line no-shadow
+  tree.addEventListener('click', (event) => {
```

Comment:
Consider adding a comment to explain that this event listener is used to toggle the visibility of child elements when a span is clicked.